self-api/src/main/java/com/selfxdsd/api/ApiToken.java

```diff
@@ -0,0 +1,29 @@
+package com.selfxdsd.api;
+
+import java.time.LocalDateTime;
+
+/**
+ * API Token.
+ */
+public interface ApiToken {
+    /**
+     * The name of token.
+     *
+     * @return String.
+     */
+    String name();
+
+    /**
+     * Secret key.
+     *
+     * @return Byte array.
+     */
+    byte[] secret();
```

Comment:
@andreoss Just make it a String and name this method ``token()`` or ``value()``

Reply:
@amihaiemil Ok, btw `byte[]` was used because the token can be sensitive information